src/thor/costmatrix.cc

```diff
@@ -16,11 +16,12 @@ namespace {
 
 constexpr uint32_t kMaxMatrixIterations = 2000000;
 constexpr uint32_t kMaxThreshold = std::numeric_limits<int>::max();
+constexpr uint32_t kMaxLocationCache = 20;
 
 // Find a threshold to continue the search - should be based on
 // the max edge cost in the adjacency set?
 int GetThreshold(const travel_mode_t mode, const int n) {
-  return (mode == travel_mode_t::kDrive) ? std::min(2700, std::max(100, n / 3)) : 500;
+  return (mode == travel_mode_t::kDrive) ? std::min(2800, std::max(100, n / 3)) : 500;
```

Comment:
more expansion now, so we should increase the maximum a bit. at least I had a negative diff because of this. eventually we should do the TODO though maybe..